lib/rules/media-query-no-invalid/index.mjs

```diff
@@ -62,7 +65,12 @@ const rule = (primary) => {
 
 						if (isCustomMediaQuery(name)) {
 							// In a plain context, custom media queries are invalid.
-							invalidNodes.push(parent);
+							complain(
```

Comment:
nit

Since you are reusing the same arguments 5 times you could maybe set a new function at l65.
e.g.
```
const foo = complain.bind(bar, atRule, atRuleParamIndexValue, parent);
[…]
foo('custom media queries can only be used in boolean queries');
```

Reply:
[suggest] Good point. How about reducing duplications like this? 👇🏼 

```js
const reasons = {
	custom: 'custom media queries can only be used in boolean queries',
	min_max_in_range: '"min-" and "max-" prefixes are not needed when using range queries',
	min_max_in_boolean: '"min-" and "max-" prefixes are not needed in boolean queries',
	discrete: 'discrete features can only be used in plain and boolean queries',
};

// ...

complain(atRule, atRuleParamIndexValue, parent, 'custom');

// ...

/**
 * @param {keyof reasons} [reason]
 */
function complain(atRule, index, node, reason) {
	report({
		// ...
		messageArgs: [node.toString(), reason ? reasons[reason] : '']
		// ...
	});
}
```